alerter.c

```diff
@@ -1,31 +1,65 @@
-#include <stdio.h>
-#include <assert.h>
+#include "alerter.h"
 
 int alertFailureCount = 0;
 
 int networkAlertStub(float celcius) {
-    printf("ALERT: Temperature is %.1f celcius.\n", celcius);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+    

```

Comment:
Test stub should be used to effectively test production code. the No-Op code here does not help.
Varying output of test stub should help us to different flow execution in production code.